src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Study.java

```diff
@@ -66,6 +66,7 @@
     @ManyToOne(optional = false)
     private Taxonomy taxonomy;
 
+    @NotNull
     @JsonProperty
```

Comment:
I think we can add  @JsonProperty(defaultValue = "false")

Reply:
OK! I have reverted my last commit for adding `@NotNull` here. The annotation does not work with primitive types.
Because this field `deprecated` is a `boolean`, a primitive type so it has default value as `false` anyway.
The `@JsonProperty(defaultValue = "false")` does not set the default value for the field. It is there mostly for "documentation" purpose only, as explained in https://fasterxml.github.io/jackson-annotations/javadoc/2.9/com/fasterxml/jackson/annotation/JsonProperty.html#defaultValue()

So I am happy with `boolean deprecated`. But I can add `defaultValue = "false"` to `@JsonProperty` for better documentation.